src/main.css

```diff
@@ -1,3 +1,16 @@
-body {
-  background-color: red;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+main {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.menuDiv {
+  width: 25%;
+  margin: 0 4% 0 4%;
```

Comment:
![image](https://user-images.githubusercontent.com/45668340/132262698-8d0b5688-0799-4fe3-b1ee-7b1568fad2c5.png)
While you made a frantic effort to smartly hit the API and display the result on the page (Desktop), I think it would be of great benefit to us if you follow the principle of "Mobile First" development strictly. The mobile view needs to be worked on a little bit, I think.